src/redux/configureStore.js

```diff
@@ -1,30 +1,12 @@
-import { combineReducers, configureStore } from '@reduxjs/toolkit';
-import { persistReducer, persistStore } from 'redux-persist';
-import storage from 'redux-persist/lib/storage';
-import { reducer as formReducer } from 'redux-form';
-import bookReducer, { getBooks } from './Books/Books';
-import categoriesReducer, { getCategories } from './categories/categories';
+/* eslint-disable */
+import { combineReducers, configureStore } from "@reduxjs/toolkit";
+import bookReducer from "./Books/Books";
+import categoriesReducer from "./categories/Categories";
```

Comment:
- [ ] There is also a problem with the naming of `Categories` here as well, so an error occurs because of that fact. Kindly fix it.
![image](https://user-images.githubusercontent.com/46839548/194675835-158b1eb9-ad06-431a-9845-dea305a3376d.png)